contribs/gmf/less/vars.less

```diff
@@ -22,7 +22,7 @@
 @above-search-index: 6;
 
 
-@nav-width: 26rem;
+@nav-width: 400px;
```

Comment:
Because `26rem` doesn't mean anything for a panel width. `rem`s should be use for fonts or typography only IMHO.